packages/shared/src/graphql/squads.ts

```diff
@@ -85,7 +86,7 @@ export const CREATE_SQUAD_MUTATION = gql`
   }
 `;
 
-export const ADD_POST_TO_SQUAD_MUTATION = gql`
+export const ADD_POST_TO_SQUAD_MUTATION = gql`x
```

Comment:
I think there was a typo.
```suggestion
export const ADD_POST_TO_SQUAD_MUTATION = gql`
```